README.md

```diff
@@ -108,7 +108,7 @@ To use in rails, Add dependency to Gemfile.
 Then invoke install task.
 
 ```
-bin/rails glueby:contract:install
+bin/rails glueby:contract:initializer
```

Comment:
`glueby:contract:install` is correct. check it using `rails --tasks`.

Reply:
Sorry I confused.
`glueby:contract:install` is corect in `rails` tasks.
but, `glueby:contract:innitializer` is correct in `rails generate` command.

my rails g --help result:
```
nakajo@ip-192-168-26-39 workshop % bundle exec rails g --help                  
Running via Spring preloader in process 74643
Usage: rails generate GENERATOR [args] [options]

General options:
  -h, [--help]     # Print generator's options and usage
  -p, [--pretend]  # Run but do not make any changes
  -f, [--force]    # Overwrite files that already exist
  -s, [--skip]     # Skip files that already exist
  -q, [--quiet]    # Suppress status output

Please choose a generator below.

Rails:
  application_record
  assets
  benchmark
  channel
  controller
  generator
  helper
  integration_test
  jbuilder
  job
  mailbox
  mailer
  migration
  model
  resource
  scaffold
  scaffold_controller
  system_test
  task

ActiveRecord:
  active_record:application_record

Glueby:
  glueby:contract:block_syncer
  glueby:contract:initializer
  glueby:contract:reissuable_token
  glueby:contract:timestamp
  glueby:contract:wallet_adapter

RackProfiler:
  rack_profiler:install

TestUnit:
  test_unit:channel
  test_unit:generator
  test_unit:install
  test_unit:mailbox
  test_unit:plugin

```